docs/the_black_code_style/future_style.md

```diff
@@ -1,117 +1,38 @@
 # The (future of the) Black code style
 
-```{warning}
-Changes to this document often aren't tied and don't relate to releases of
```

Comment:
This warning isn't really true any more now that we explicitly list preview features.